recipes/cuda-cudart/meta.yaml

```
@@ -0,0 +1,290 @@
+  sha256: c0e16e9593a69afe4997ae96a8b024e751ebce0254523fd969b08bb028582d23  # [linux64]
+  sha256: d785fd613e35e3761fdb0016a31a56417f570605cbfb9a93ca9dc68be6789188  # [ppc64le]
+  sha256: 5de248c6c203f6c91ce34256bbfc028bc87c1b0c552f9f51edb65e60ad665f63  # [aarch64]
+  sha256: 1cd8516feee068c6d718453e76b0dfcbee27844a35d4f3aa608a3d316190309c  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx or win]
+
+outputs:
+  - name: cuda-cudart
+    files:
+      - lib/libcu*.so.*                            # [linux]
+      - targets/{{ target_name }}/lib/libcu*.so.*  # [linux]
+      - Library\bin\cudart*.dll                    # [win]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - sysroot_{{ target_platform }} 2.17    # [linux]
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+    test:
+      commands:
+        - test -L $PREFIX/lib/libcudart.so.{{ version }}                                          # [linux]
+        - test -L $PREFIX/lib/libcudart.so.{{ version.split(".")[0] }}                            # [linux]
+        - test -L $PREFIX/targets/{{ target_name }}/lib/libcudart.so.{{ version }}                # [linux]
+        - test -f $PREFIX/targets/{{ target_name }}/lib/libcudart.so.{{ version.split(".")[0] }}  # [linux]
+        - if not exist %LIBRARY_BIN%\cudart64_{{ version.split(".")[0] }}.dll exit 1              # [win]
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
+      license_file: LICENSE
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: CUDA Runtime native Libraries
+      description: |
+        CUDA Runtime native Libraries
+      doc_url: https://docs.nvidia.com/cuda/index.html
+
+  - name: cuda-cudart-dev
+    build:
+      run_exports:
+        - {{ pin_subpackage("cuda-cudart", max_pin="x") }}
+    files:             # [linux]
+      - lib/libcu*.so  # [linux]
+      - lib/pkgconfig  # [linux]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - sysroot_{{ target_platform }} 2.17    # [linux]
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+        - {{ pin_subpackage("cuda-cudart", exact=True) }}
+        - {{ pin_subpackage("cuda-cudart-dev_" + target_platform, exact=True) }}
+        - {{ pin_subpackage("cuda-cudart-static", exact=True) }}
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}                    # [aarch64]
```

Normalize the whitespace to match the other lines like this.
```suggestion
        - arm-variant * {{ arm_variant_type }}  # [aarch64]
```